server/api/admin/auth/signin.post.ts

```diff
@@ -0,0 +1,66 @@
+//ADMIN SIGN INTO THEIR ACCOUNT
+
+import generateToken from "../../../plugins/generateToken";
+import { Users } from "../../../dbModels";
+
+interface IRequestBody {
+	email: string;
+	password: string;
+}
+export default defineEventHandler(async (e) => {
+	console.log("POST /api/auth/signin");
+	const { email, password } = await useBody<IRequestBody>(e); // Check if email is passed.
+	if (!email) {
+		e.res.statusCode = 400;

```

Comment:
request body can be checked by validators or through middlewares.

an example validator package - https://joi.dev/api/?v=17.6.1

Reply:
Sure. I will try to implement that in the code.